app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -370,8 +370,6 @@ ShortFormApplicationController = (
   $scope.checkAfterLiveInTheNeighborhood = (preference) ->
     # preference is either neighborhoodResidence or antiDisplacement
     if ShortFormApplicationService.applicationHasPreference(preference)
-      # Neighborhood provides automatic liveInSf preference
-      ShortFormApplicationService.copyNeighborhoodToLiveInSf(preference)
```

Comment:
@kaptron  this is the redundant function i suggested to remove from my first comment.